elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -269,6 +275,24 @@ class PfeTabs extends PFElement {
 
     this._linked = false;
     this._linkPanels();
+
+    if (mutationsList) {
```

Comment:
@kylebuch8 This should be handled by cascade now

Reply:
@castastrophe - The cascade doesn't work for items that are dynamically added. It only fires on attribute changes so we'll need to keep that mutationList part of the code in there.